pkgs/applications/editors/jupp/default.nix

```diff
@@ -2,17 +2,18 @@
 , ncurses, gpm
 }:
 
+with stdenv.lib; 
```

Comment:
Same comment as I made on your `mksh` PR: why was this moved? Doesn't seem necessary.

Reply:
It makes the `stdenv.lib` functions available to all expression, not only `meta` part.

Reply:
Yes, but there's no need for `stdenv.lib` to be available anywhere else but the `meta` block, AFAICS.